pkg/build/archive.go

```diff
@@ -124,6 +127,30 @@ func Archive(manifest model.Manifest) error {
 	return nil
 }
 
+// sanitizeHelmCharts changes the hub and tag for all helm charts in archive, except
+// the operator
+func sanitizeHelmCharts(manifest model.Manifest, out string) error {
```

Comment:
should we just find all **/values.yaml?

Reply:
I added back the capability in 1.4 which listed out the charts. It's not quite that elegant, but I figure it worked before. I can walk the tree if you think that's easier.